tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/XPathAlt.java

```diff
@@ -0,0 +1,94 @@
+package org.unicode.cldr.web.api;
+
+import java.util.*;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
+import org.eclipse.microprofile.openapi.annotations.media.Content;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import org.eclipse.microprofile.openapi.annotations.parameters.Parameter;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponses;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+import org.unicode.cldr.icu.LDMLConstants;
+import org.unicode.cldr.util.*;
+import org.unicode.cldr.web.CookieSession;
+
+@Path("/alt")
```

Comment:
```suggestion
@Path("/xpath/alt")
```

Reply:
should that be true of /api/whatis as well? --> /api/xpath/whatis (then also need to revise in LookUp.vue)

Reply:
for now I've just fixed it for alt

Reply:
probably good to move whatis also.